MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -198,6 +199,20 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>payara5.194+</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>/5\.(194|[2-9]\d\d)(?:-SNAPSHOT|\.\d*)?/</value>
+                </property>
+            </activation>
+            <properties>
+                <microprofile.rest.client.tck.version>1.3.4</microprofile.rest.client.tck.version>
```

Comment:
Can we have the latest as the main property please, and defer to older versions in the specific "old" profiles

Reply:
Figured out that this is not possible, as for rest client 192+ we're adding dependencies that are not necessary or needed for 191.